nucypher/blockchain/eth/deployers.py

```diff
@@ -467,7 +467,7 @@ def __check_policy_manager(self):
             raise RuntimeError("PolicyManager contract is not initialized.")
 
     def _deploy_essential(self, gas_limit: int = None):
-        escrow_constructor_args = (self.token_contract.address, *self.economics.staking_deployment_parameters)
+        escrow_constructor_args = (self.token_contract.address, *self.economics.staking_deployment_parameters, False)
```

Comment:
Heh - this lonely `False` is the worklock test flag?

Reply:
Yep, and it look ugly, don't know how to do it better

Reply:
Perhaps the value can be an Init parameter to the Deployer instance itself  (`test_mode`) - this flag is lacking any naming within the python API, which I think will contribute to it being forgotten / neglected.

Reply:
✔️

Reply:
OK, I'm in - but I do think we may need to control this parameter via CLI flags, and make a section about it in our deployment documentation so that it is not forgotten.